### PR TITLE
New Feature #5309: Multiple entries in pickList

### PR DESCRIPTION
**Pull request by e-reznik**
I'm trying to implement the new feature request of [allowing multiple entries in the PickList](https://github.com/primefaces/primefaces/issues/5309).
So far, I've just added the new attribute _allowMultiple_, without any functionality yet.

As this is my first attempt to contribute to such a huge project, every guidance is appreciated :relaxed:

### CODE REVIEW

**Comment by melloware** [comment]
Good start. How about `allowDuplicates` as more accurate?

**Comment by e-reznik** [comment]
@melloware: Thanks, I changed the name.

File: src/main/java/org/primefaces/component/picklist/PickListBase.java [hunk 1]
```diff
@@ -99,6 +100,14 @@ public void setVar(String var) {
         getStateHelper().put(PropertyKeys.var, var);
     }
 
+    public String getAllowDuplicates() {
```
Comment (melloware) [inline src/main/java/org/primefaces/component/picklist/PickListBase.java]: This should return `boolean` and be `isAllowDuplicates`

**Review by melloware — CHANGES_REQUESTED** [review]

File: src/main/java/org/primefaces/component/picklist/PickListBase.java (hunk 1 above)
Comment (e-reznik) [reply src/main/java/org/primefaces/component/picklist/PickListBase.java]: My bad. Fixed.

**Comment by melloware** [comment]
Looks good to continue

**Comment by tandraschko** [comment]
I will close it for now for clarity of our PR list.
Feel free to create a PR again when you finished the implementation :) 

Thanks!

**Pull request closed without merging**